### ISSUE TITLE
Calling setInterval in server side component

### ISSUE THREAD

**Issue by bartaz** [canonical-ols/build.snapcraft.io#96]
In builds listing container, on component mount we initialise polling to fetch builds listing.
On client side it properly initialises polling and disables it on unmount.

```js
  componentWillMount() {
    this.fetchData(this.props);

    this.fetchInterval = setInterval(() => {
      this.fetchData(this.props);
    }, 15000);
  }
```
from: [builds.js](https://github.com/canonical-ols/build.snapcraft.io/blob/master/src/common/containers/builds.js#L24-L30)

But when page is rendered on server it is also initialised and it seems that it's never disabled - component is not unmounted, interval keeps running and multiple instances of it are created (one for each serverside request for this page).

Because server doesn't have all data needed to fetch the data nothing happens (no API calls are made and component is just rendered empty), but callbacks keep running.

We should either initialise polling on client side only, or make sure once rendered, components are properly unmounted on server side so that interval will be cleared (before it's even called).

**Comment by earnubs** [comment]
I vote for cleint side only, ComponentDidMount? https://facebook.github.io/react/docs/react-component.html#componentdidmount

**Comment by bartaz** [comment]
Haven't thought about that. Yes, `componentDidMount` is not called on server side, so it nicely fixes the issue.

Addressed already in #91

**Comment by bartaz** [comment]
Fixed by #91
